src/agent/driver-test-runner.js

```diff
@@ -90,32 +90,35 @@ export class DriverTestRunner {
     await this.atDriver.sendKeys(sequence);
   }
 
+  /**
+   *
+   * @param {"reading" | "interactive" | undefined} mode
```

Comment:
If `mode` may be undefined, then we can't call `String.prototype.toLowerCase` on it, below.

Reply:
good point - I'm filtering out the undefined at the call site anyway, so just made the assumption "reading" and "interactive" for now -- v2 this changes to a "settings" setup I think